codice/ddf
# DDF-2414: Create an OWASP runner that runs only on changed poms

#### What does this PR do?
@ryeats @adimka
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@figliold 
#### How should this be tested?
Pull down latest master. Add a commit that adds new lines to several poms. Do a clean install in the owasp-diff-runner module using the `-Powasp-diff-runner` flag and ensure it runs owasp against the changes poms and builds successfully. Next add a known vulnerability with a CVS of 7 or higher to a pom and build the module again with the `-Powasp-diff-runner` flag. This build should fail on OWASP from OWASP findings.
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Checklist:
